src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -329,12 +341,18 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 			const elementBox = this.element.getBoundingClientRect();
 			contextMenuService.showContextMenu({
 				getActions: () => {
+					const contextKeyService = this._contextKeyService!.createOverlay([
+						...Object.entries(data.context),
+						['webview', this.providedId],
```

Comment:
I will look into adding a generic `activeWebviewId` context key that is set whenever a webview is focused. This matches the `activeCustomEditorId` context key we already have

Reply:
Is that different than `webview`? That is currently the "id" of the webview that the context menu is being shown for

Reply:
Or are you saying to usages outside of the context menu -- a more global context key?